debian/debian/ix-reporting.service

```diff
@@ -0,0 +1,14 @@
+[Unit]
+Description=Setup TrueNAS Reporting
+DefaultDependencies=no
+
+After=middlewared.service network-online.target httpd.service squid.service nfs-server.service mysqld.service named.service postfix.service smartmontools.service nut.target
```

Comment:
What is going on here? This is a ton of lines in `After`?

Reply:
@yocalebo we are replicating `netdata` service's `After` here

Reply:
Okay, please put a comment in this file describing this fact.